src/Tabs.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class Tabs extends React.Component {
+  state = {
+    tabNumber: 0,
+  };
+
+  tabSwitcher = (event) => {
+    this.setState({
+      tabNumber: +event.target.value,
+    });
+  };
+
+  render() {
+    const { tabs: tabsR } = this.props;
+    const { tabNumber } = this.state;
+
+    return (
+      <>
+        <button
+          type="button"
+          value="0"
+          className={tabNumber === 0 ? 'active' : ''}
+          onClick={this.tabSwitcher}
+        >
+          Tab 1
+        </button>
```

Comment:
I'm not sure but guess it would be better to create your buttons in a loop, you can bind values to index and the name of the button is index + 1, and then you can avoid DRY)

Reply:
good advice